xarray/tests/test_missing.py

```diff
@@ -117,7 +117,7 @@ def test_interpolate_pd_compat():
 
 
 @requires_scipy
-@pytest.mark.parametrize("method", ["barycentric", "krog", "pchip", "spline", "akima"])
+@pytest.mark.parametrize("method", ["barycentric", "krogh", "pchip", "spline", "akima"])
 def test_scipy_methods_function(method):
```

Comment:
```suggestion
def test_scipy_methods_function(method) -> None:
```

Reply:
@Illviljan Thanks for reminding me to check for missing typing. How about the `method` argument. Does it need typing too?

Reply:
You could import InterpolantOptions if you want. Does mypy complain if you add a misspelled method without any typing?